src/loops/programs_with_repetition/easy/q5/MirrorSeven.java

```diff
@@ -0,0 +1,42 @@
+package loops.programs_with_repetition.easy.q5;
+
+/*
+ * Write a program to keep asking the user to enter a number until they enter a palindrome number that is also
+ * divisible by 7 (i.e. it can be read the same forwards and backwards)
```

Comment:
```suggestion
 * divisible by 7 (i.e. it can be read the same forwards and backwards). Once it receives a valid integer, the program should print the message 
```